### ISSUE TITLE
Standardize logits in MoT

### ISSUE THREAD

**Issue by nrfulton** [generative-computing/mellea#123]
1. Add a `ModelOption.LOGITS` for determining whether we should pass back logits.
2. The `ModelOutputThunk` should pass back logits if `ModelOption.LOGITS=True` and also logits are provided by the inference engine.
